coordinator/src/orderbook/trading.rs

```diff
@@ -293,6 +294,27 @@ async fn process_pending_match(
         if let Err(e) = notify_trader(trader_id, message, authenticated_users).await {
             tracing::warn!("Failed to notify trader. Error: {e:#}");
         }
+    } else if let Some(position) =
+        positions::Position::get_open_position_by_trader(conn, trader_id.to_string())?
+    {
+        tracing::debug!(%trader_id, position_id=position.id, "Checking if the users positions is eligible for rollover");
+
+        if orderbook_commons::is_in_rollover_weekend(position.expiry_timestamp) {
```

Comment:
because we are currently managing the users in the trading component.. that should probably not be the case. however, I haven't really thought about it. I guess you are right, eventually the orderbook should not care for the technicality of rolling over a dlc.

Reply:
I think that would require a refactoring on how we are managing users currently connected to our websocket. Not sure if we want to tackle this already with that PR.